docs/design/semantic_slots.md

```diff
@@ -29,12 +29,12 @@ In the table below, you can see all eight palette colors defined for the bodyTe
 
 ![Example table showing semantic slots on light and dark theme variants](../images/doc-semantic-slot-940px-table.png)
 
-|         |      Light themes      |      Dark themes       |
-| ------- | ---------------------- | ---------------------- |
-| None    | neutralPrimary #333333 | neutralPrimary #ffffff |
+| | Light themes | Dark themes |

```

Comment:
Unnecessary changes to this file, please revert. These changes make the table harder to read.

Reply:
Old content re-added.